### ISSUE TITLE
Mention `MIX_ENV=prod mix compile` earlier in the deployment guide

### ISSUE THREAD

**Issue by nathanhornby** [phoenixframework/phoenix_guides#467]
`MIX_ENV=prod mix compile` (along with the `--only prod` flag for `deps.get`) is mentioned as a step in the summary at the end of the guide, but not as part of the guide itself.

It could probably be mentioned just before "Compiling your applications assets" very briefly.

```
# Initial setup
$ mix deps.get --only prod
$ MIX_ENV=prod mix compile
```

**Comment by jeregrine** [comment]
@nathanhornby could you pr this change? Thanks!

**Comment by ybur-yug** [comment]
@jeregrine I'll take a stab at this today, I'm peeking at a few other documentation related tasks already :)

**Comment by ybur-yug** [comment]
@jeregrine addressed in #511

**Comment by ybur-yug** [comment]
@nathanhornby @jeregrine I think this is good to close now that its been merged.
Is there a convention or regular flow for the whole `issue opened -> PR addresses it -> PR merged` setup with regard to closing the loose ends on the original issue opened, or is it just done manually once someone has the time?

Not pushing or anything, just curious and wanted to make sure I matched how you guys did it if there was already a standard :)
